README.rst

```diff
@@ -145,3 +145,28 @@ Example:
         def test_special_method(self):
             with self.record_performance():
                 list(Author.objects.special_method())
+
+Settings
+========
+
+Behaviour for **django-perf-rec** may be customized with a settings dictionary
+in Django's ``settings.py`` file:
```

Comment:
I normally just say 'Django settings' because most projects don't use a single settings file in the end